ios/FluentUI/Command Bar/CommandBarTokens.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class MSFCommandBarTokens: MSFTokensBase {
+    public var backgroundColor: UIColor!
+    public var iconColor: UIColor!
+
+    public var hoverBackgroundColor: UIColor!
+    public var hoverIconColor: UIColor!
+
+    public var pressedBackgroundColor: UIColor!
+    public var pressedIconColor: UIColor!
+
+    public var selectedBackgroundColor: UIColor!
+    public var selectedIconColor: UIColor!
+
+    public var disabledBackgroundColor: UIColor!
+    public var disabledIconColor: UIColor!
+
+    public var barBackgroundColor: UIColor!
+    public var fixedIconColor: UIColor!
+    public var groupBorderRadius: CGFloat!
+    public var groupInterspace: CGFloat!
+    public var iconSize: CGFloat!
+    public var itemInterspace: CGFloat!
+
+    override init() {
+        super.init()
+
+        self.themeAware = true
+
+        updateForCurrentTheme()
+    }
+
+    @objc open func didChangeAppearanceProxy() {
+        updateForCurrentTheme()
```

Comment:
I just noticed this issue where the primary brand color is not updated in the keyboard CCB:

https://user-images.githubusercontent.com/68076145/134248265-2340c2bf-725d-46ab-ae5f-652f51c6c66b.mov

Reply:
Good find! Isn't the keyboard in its own `UIWindow` instance? If the accessory bar is attached to that, then it may not have the same root window, and thus has a separate theme.

Reply:
@mischreiber yes, that's what I suspected too. And it does repro on the current CCB (before tokenization).
@sophialee0416 if this is the case, It'd be good to set the ColorProviding theme for the keyboard window so we have a good sample code for the client apps.

Reply:
Resolved offline: will create a separate task for theming mismatch due to different UIWindows as a part of the theming initiative @mischreiber @rdeassis